### ISSUE TITLE
[Bug]: Qwen W8A8 example does not quantize down projection

### ISSUE THREAD

**Issue by learning-chip** [vllm-project/vllm-ascend#979]
### Your current environment

vLLM Version: 0.8.5.post1
vLLM Ascend Version: 0.8.5rc1

### 🐛 Describe the bug

Profiling of https://www.modelscope.cn/models/vllm-ascend/Qwen2.5-0.5B-Instruct-w8a8 or https://www.modelscope.cn/models/vllm-ascend/Qwen3-8B-w8a8 shows that the down projection in MLP (`RowParallelLinear`) is not using quant_matmul, but still uses `aten::matmul`:

![Image](https://github.com/user-attachments/assets/ea034698-691b-46d3-96ea-f3ad4eb99b48)

To reproduce:

```python
import torch
import torch_npu
from vllm import LLM, SamplingParams

model_path = "/model_weights/"Qwen2.5-0.5B-Instruct-w8a8"
# same for Qwen3-8B-w8a8

llm = LLM(
    model=model_path,
    max_model_len=2048,
    max_num_seqs=16
)

prompts = [
    "Hello, my name is",
    "The future of AI is",
]
sampling_params = SamplingParams(
    max_tokens=3, temperature=0.8, top_p=0.95, seed=0)

log_dir = 'profile_trace'
with torch_npu.profiler.profile(
        on_trace_ready=torch_npu.profiler.tensorboard_trace_handler(log_dir),
        activities=[torch_npu.profiler.ProfilerActivity.CPU, torch_npu.profiler.ProfilerActivity.NPU],
        record_shapes=True,
        profile_memory=True,
        with_flops=True,
        with_stack=True
) as prof:
    outputs = llm.generate(prompts, sampling_params)
```

**Comment by Yikun** [comment]
@learning-chip Much thanks for report.

@22dimensions Please take a look

**Comment by leo-pony** [comment]
Bug has been fixed, now you can try with v0.9.1 or v0.11.0rc1.
Using guide:
https://docs.vllm.ai/projects/ascend/en/v0.9.1/user_guide/feature_guide/quantization.html
https://docs.vllm.ai/projects/ascend/en/latest/user_guide/feature_guide/quantization.html

**Comment by zhaomingyu13** [comment]
@learning-chip This issue has been resolved in the latest version. Please use the latest release or rc version (https://github.com/vllm-project/vllm-ascend/releases). If the issue persists, feel free to submit a new issue.

**Comment by kunpengW-code** [comment]
I checked the `quant_model_description.json` files of these two models, and the `down_proj` part is not quantized. For example, `"model.layers.0.mlp.down_proj.weight": "FLOAT"`. Therefore, this should not be an issue, but rather the model weights themselves do not have this part quantized. The reason for this is likely to ensure accuracy.
